bumblebee/modules/datetime.py

```diff
@@ -15,6 +15,11 @@
 import datetime
 import locale
 import bumblebee.engine
+import os
+try:
+    import pytz
+except ModuleNotFoundError:
```

Comment:
This is Python 3 only and broke my installation of `bumblebee-status` in a system in which `usr/bin/env` points to Python 2. Can we change [this line](https://github.com/tobi-wan-kenobi/bumblebee-status/blob/master/bumblebee-status#L1) to `/usr/bin/env python3` and stop supporting Python 2 given that it is soon going to be [deprecated tomorrow](https://www.python.org/doc/sunset-python-2/)?